partner-api.yaml

```
@@ -916,31 +476,187 @@ paths:
           description: Forbidden
         '404':
           description: Not Found
-      description: 'Ermittelt, ob der Partner auf die Vorgänge eines anderen Partner zugreifen darf'
-      parameters:
-        - schema:
-            type: string
-          in: header
-          name: X-PartnerId
-          required: true
-        - schema:
-            type: string
-          in: header
-          name: X-ApiKey
-          required: true
-        - schema:
-            type: string
-          in: header
-          name: X-TraceId
-        - schema:
-            type: string
-            default: application/json
-          in: header
-          name: Accept
-          description: ''
+
 components:
+  securitySchemes:
+    europace_oauth2:
+      type: oauth2
+      description: This API uses OAuth 2 with the client credentials flow. [More info](https://auth0.com/docs/flows/concepts/client-credentials)
+      flows:
+        clientCredentials:
+          tokenUrl: https://api.europace.de/auth/token
+          scopes:
+            partner:entity:create: Create a new Partner entity
+            partner:entity:delete: Delete the whole Partner
+            partner:entity:read: Read partner data
+            partner:entity:write: Modify data of a partner
+            partner:relationships:read: Read relationships between partners
+            partner:relationships:write: Modify relationships between partners
+            partner:rechte:read: Read Rechte eines Partners
+            partner:rechte:write: Schreibe Rechte eines Partners
```

Ich bin der Meinung, wir sollten den "nackten" Scope in der UI anzeigen, plus die Beschreibung dazu. Ich denke das wird letztendlich unser Leben einfacher machen und ist ein valider erster Schritt. Wenn es die Nutzer zu sehr verwirrt, können wir immer noch "human-readable" Scope nahmen einführen.

Alle Scopes auf Deutsch:
```suggestion
            partner:plakette:anlegen: Darf neue Plaketten anlegen.
            partner:plakette:löschen: Darf Plaketten löschen
            partner:plakette:lesen: Darf Partner-Daten lesen
            partner:plakette:schreiben: Darf Partner-Daten schreiben
            partner:beziehungen:lesen: Darf Beziehungen zwischen Partnern lesen
            partner:beziehungen:schreiben: Darf Beziehungen zwischen Partnern schreiben
            partner:rechte:lesen: Darf Rechte eines Partners lesen
            partner:rechte:schreiben: Darf Rechte eines Partners schreiben
```

---

Eine frage noch: warum gibt es den Scoe "löschen"? Ist damit Sperren gemeint?

---

Nein, man kann per API auch Partner komplett löschen